package.json

```diff
@@ -6,14 +6,12 @@
   "repository": "schibsted/eslint-config-schibsted",
   "license": "MIT",
   "scripts": {
-    "preinstall": "npm run install:config && npm run install:config-react",
-    "install:config": "cd packages/eslint-config-schibsted && npm prune && npm install",
-    "install:config-react": "cd packages/eslint-config-schibsted-react && npm prune && npm install",
-    "pretest": "npm link packages/eslint-config-schibsted",
-    "test": "npm run test:schibsted && npm run test:schibsted-react",
-    "posttest": "npm unlink packages/eslint-config-schibsted",
-    "test:schibsted": "cd packages/eslint-config-schibsted && npm test",
-    "test:schibsted-react": "cd packages/eslint-config-schibsted-react && npm test"
+    "install": "npm run install:eslint-config-schibsted && npm run install:eslint-config-schibsted-react",
+    "install:eslint-config-schibsted": "cd packages/eslint-config-schibsted && yarn",
```

Comment:
Thought it'll be more expressive if we mention the full name of the package. But maybe we can stick with the "eslint" package name (that's without `eslint-config-` prefix. Is that what you're aiming for? I'm cool with that.